doc.nix

```diff
@@ -6,20 +6,20 @@ let
     rootMountPoint = "/mnt";
   };
   eval = lib.evalModules {
-    modules =  [
+    modules = [
       {
         options.disko = {
           devices = lib.mkOption {
             type = types.devices;
-            default = {};
+            default = { };
             description = "The devices to set up";
           };
         };
       }
     ];
   };
   options = nixosOptionsDoc {
-    options = eval.options;
+    inherit (eval) options;
```

Comment:
I think @Lassulus didn't liked those `inherit` constructs for readability reasons. If you could skip those in statix than it's good to merge.

Reply:
We can merge them with the inherits and can try to configure the formater later